2025-08-30 21:05:38 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_login_info

2025-08-30 21:05:38 | ERROR    | src.core.bot:get_login_info:97 - 无法获取登录信息: 没有可用的WebSocket连接

2025-08-30 21:05:50 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:05:50 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:05:55 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: send_group_msg

2025-08-30 21:05:55 | ERROR    | src.core.bot:send_group_msg:77 - 无法发送群消息: 没有可用的WebSocket连接

2025-08-30 21:06:00 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:06:00 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:06:05 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: send_group_msg

2025-08-30 21:06:05 | ERROR    | src.core.bot:send_group_msg:77 - 无法发送群消息: 没有可用的WebSocket连接

2025-08-30 21:06:10 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:06:10 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:06:15 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:06:15 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:06:20 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:06:20 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:06:25 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:06:25 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:06:30 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:06:30 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:06:35 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:06:35 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:06:59 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:06:59 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:07:04 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: send_group_msg

2025-08-30 21:07:04 | ERROR    | src.core.bot:send_group_msg:77 - 无法发送群消息: 没有可用的WebSocket连接

2025-08-30 21:07:09 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:07:09 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:07:14 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:07:14 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:07:19 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:07:19 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:07:24 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:07:24 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:07:43 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:07:43 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:07:51 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:07:51 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:07:56 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:07:56 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:08:28 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:08:28 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:08:33 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:08:33 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:08:38 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:08:38 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:09:26 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:09:26 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:09:43 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:09:43 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:09:48 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:09:48 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:09:59 | ERROR    | src.core.bot:send_websocket_message:177 - [OneBot] API请求超时: get_group_info

2025-08-30 21:09:59 | ERROR    | src.core.bot:get_group_info:123 - 无法获取群信息: 没有可用的WebSocket连接

2025-08-30 21:15:13 | ERROR    | src.core.bot:_make_request:77 - [OneBot] API请求异常: get_login_info, 错误: Cannot connect to host 127.0.0.1:8029 ssl:default [远程计算机拒绝网络连接。]

2025-08-30 21:15:30 | ERROR    | src.core.bot:_make_request:77 - [OneBot] API请求异常: get_group_info, 错误: Cannot connect to host 127.0.0.1:8029 ssl:default [远程计算机拒绝网络连接。]

2025-08-30 21:15:46 | ERROR    | src.core.bot:_make_request:77 - [OneBot] API请求异常: get_group_info, 错误: Cannot connect to host 127.0.0.1:8029 ssl:default [远程计算机拒绝网络连接。]

2025-08-30 21:15:48 | ERROR    | src.core.bot:_make_request:77 - [OneBot] API请求异常: send_group_msg, 错误: Cannot connect to host 127.0.0.1:8029 ssl:default [远程计算机拒绝网络连接。]

2025-08-30 21:16:25 | ERROR    | src.core.bot:_make_request:77 - [OneBot] API请求异常: get_group_info, 错误: Cannot connect to host 127.0.0.1:8029 ssl:default [远程计算机拒绝网络连接。]

2025-08-30 21:16:35 | ERROR    | src.core.bot:_make_request:77 - [OneBot] API请求异常: get_group_info, 错误: Cannot connect to host 127.0.0.1:8029 ssl:default [远程计算机拒绝网络连接。]

2025-08-30 21:16:47 | ERROR    | src.core.bot:_make_request:77 - [OneBot] API请求异常: get_group_info, 错误: Cannot connect to host 127.0.0.1:8029 ssl:default [远程计算机拒绝网络连接。]

2025-08-30 21:16:49 | ERROR    | src.core.bot:_make_request:77 - [OneBot] API请求异常: send_group_msg, 错误: Cannot connect to host 127.0.0.1:8029 ssl:default [远程计算机拒绝网络连接。]

2025-08-30 21:17:01 | ERROR    | src.core.bot:_make_request:77 - [OneBot] API请求异常: get_group_info, 错误: Cannot connect to host 127.0.0.1:8029 ssl:default [远程计算机拒绝网络连接。]

2025-08-30 21:17:07 | ERROR    | src.core.bot:_make_request:77 - [OneBot] API请求异常: get_group_info, 错误: Cannot connect to host 127.0.0.1:8029 ssl:default [远程计算机拒绝网络连接。]

2025-08-30 21:36:20 | ERROR    | src.gui.config_window:load_config:465 - 加载配置失败: 'NoneType' object has no attribute 'onebot'

2025-08-30 21:38:10 | ERROR    | src.gui.config_window:load_config:465 - 加载配置失败: 'ServerConfig' object has no attribute 'workers'

2025-08-30 21:40:44 | ERROR    | src.gui.config_window:apply_config:484 - 保存配置失败: expected floating-point number but got "10 MB"

2025-08-30 21:42:06 | ERROR    | src.gui.config_window:apply_config:491 - 保存配置失败: expected floating-point number but got "7 days"

2025-08-30 21:45:37 | ERROR    | src.gui.config_window:apply_config:497 - 保存配置失败: ConfigManager.save_config() takes 1 positional argument but 2 were given

2025-08-31 00:38:33 | ERROR    | src.gui.main_window_qt:open_config_window:966 - 打开配置窗口失败: name 'QWidget' is not defined

2025-08-31 00:38:35 | ERROR    | src.gui.main_window_qt:open_log_window:980 - 打开日志窗口失败: expected str, bytes or os.PathLike object, not MainWindowQt

2025-08-31 00:40:14 | ERROR    | src.gui.main_window_qt:open_log_window:980 - 打开日志窗口失败: 'LogWindowQt' object has no attribute 'export_btn'

2025-08-31 00:40:16 | ERROR    | src.gui.main_window_qt:open_config_window:966 - 打开配置窗口失败: 'SiLineEdit' object has no attribute 'setEchoMode'

2025-08-31 00:41:53 | ERROR    | src.gui.main_window_qt:open_config_window:966 - 打开配置窗口失败: 'SiLineEdit' object has no attribute 'setEchoMode'

2025-08-31 00:41:55 | ERROR    | src.gui.main_window_qt:open_log_window:980 - 打开日志窗口失败: 'LogWindowQt' object has no attribute 'export_btn'

2025-08-31 00:42:50 | ERROR    | src.gui.config_window_qt:load_config:392 - 加载配置失败: 'MainWindowQt' object has no attribute 'get_all_config'

2025-08-31 00:42:51 | ERROR    | src.gui.main_window_qt:open_config_window:966 - 打开配置窗口失败: 'SiLineEdit' object has no attribute 'textChanged'

2025-08-31 00:42:53 | ERROR    | src.gui.main_window_qt:open_log_window:980 - 打开日志窗口失败: 'LogWindowQt' object has no attribute 'export_btn'

2025-08-31 00:47:35 | ERROR    | src.gui.main_window_qt:load_embedded_wordlib_list:1154 - 加载词库列表失败: name 'os' is not defined

2025-08-31 00:47:35 | ERROR    | src.gui.main_window_qt:load_embedded_config:1590 - 加载配置失败: 'ConfigManager' object has no attribute 'get'

2025-08-31 00:47:41 | ERROR    | __main__:main:108 - 程序启动失败: MainWindowQt.setup_log_viewer_embedded() takes 2 positional arguments but 3 were given

2025-08-31 00:48:47 | ERROR    | src.gui.main_window_qt:load_embedded_wordlib_list:1154 - 加载词库列表失败: name 'os' is not defined

2025-08-31 00:48:47 | ERROR    | src.gui.main_window_qt:load_embedded_config:1591 - 加载配置失败: 'ConfigManager' object has no attribute 'get'

2025-08-31 00:48:49 | ERROR    | __main__:main:108 - 程序启动失败: MainWindowQt.setup_log_viewer_embedded() takes 2 positional arguments but 3 were given

2025-08-31 00:49:35 | ERROR    | src.gui.main_window_qt:load_embedded_wordlib_list:1155 - 加载词库列表失败: expected str, bytes or os.PathLike object, not dict

2025-08-31 00:49:35 | ERROR    | __main__:main:108 - 程序启动失败: MainWindowQt.setup_log_viewer_embedded() takes 2 positional arguments but 3 were given

2025-08-31 00:54:06 | ERROR    | src.gui.main_window_qt:open_log_window:986 - 打开日志窗口失败: 'LogWindowQt' object has no attribute 'export_btn'

2025-08-31 01:25:32 | ERROR    | src.gui.main_window_qt:show_help:371 - 打开帮助窗口失败: 'SiLineEdit' object has no attribute 'setPlaceholderText'

2025-08-31 01:25:39 | ERROR    | src.gui.main_window_qt:show_help:371 - 打开帮助窗口失败: 'SiLineEdit' object has no attribute 'setPlaceholderText'

2025-08-31 02:09:13 | ERROR    | src.gui.main_window_qt:load_embedded_config:2416 - 加载配置失败: 'SiLineEdit' object has no attribute 'setText'

2025-08-31 02:10:31 | ERROR    | src.gui.main_window_qt:load_embedded_config:2416 - 加载配置失败: 'SiLineEdit' object has no attribute 'setText'

